dpmData.py

```diff
@@ -28,38 +24,52 @@ def main():
     args = parser.parse_args()
     sys.stdout.write(str(hdf_code(args)))
 
+def local_to_utc(date):
```

Comment:
Because the standard behavior in Python is to return seconds. I would name this function `local_to_utc_ms` but this isn't a needed change just wanted to point out an opportunity for best practice and clarity.